msteams-platform/toolkit/add-capability.md

```diff
@@ -96,16 +97,15 @@ After adding tab, the changes in your project are as follows:
 - The files under `templates/azure/teamsfx` will be updated, and `templates/azure/provision/xxx`.bicep file will be regenerated.
 - The file under `.fx/config` are regenerated, which ensures your project is set with right configurations for newly added capability.
 
-## Limitations
-
-The limitations with TeamsFx while adding more capabilities are as follows:
+## Limitation
 
+The limitation to TeamsFx while adding more capabilities is as follows:
 - You can't add project capability for more than one instance.
-- You can't add bot capabilities if your project contains messaging extension.
-- You can't add messaging extension if your project contains a bot.
 
 > [!NOTE]
 > If you want to include both bot and messaging extension capabilities, then select them at the same time. You can only add them either when you create a new project or a tab application.
```

Comment:
`If you want to include both bot and messaging extension capabilities, then select them at the same time.` This line is also obsolete, now you don't have to select them at the same time, you can add them later.